doc/01_overview/compliance.rst

```diff
@@ -54,6 +54,9 @@ Ibex currently supports the following features according to the RISC-V Privilege
 * Performance counters as described in :ref:`performance-counters`
 * Vectorized trap handling as described at :ref:`exceptions-interrupts`
 
+In addition Ibex has support for the Trusted Execution Environment (TEE) working group proposal `PMP Enhancements for memory access and execution prevention on Machine mode <https://docs.google.com/document/d/1Mh_aiHYxemL0umN3GTTw8vsbmzHZ_nxZXgjgOUzbvc8/edit?usp=sharing>`_.
```

Comment:
Currently it's not actually optional in that there's no parameter to remove ePMP RTL (in an attempt to avoid adding yet more config options), however you can just ignore the `mseccfg` CSR, so perhaps describe it as 'experimental' only?

Reply:
It's "optional at runtime", isn't it? That's clearly not good wording, though, but it would be nice to tell users that they can easily get the "ratified standard" behavior.

Reply:
I suppose so, I'll go for 'experimental and optional' the other doc changes you suggest will go into detail about what the 'optional' means anyway.